dev-python/python-xmp-toolkit/files/python-xmp-toolkit-2.0.1-gif87a-dropped.patch

```diff
@@ -0,0 +1,56 @@
+From 74034f55cee1239e2eeafb6a07f22a35a4f234db Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?J=C3=A9r=C3=B4me=20Carretero?= <cJ@zougloub.eu>
+Date: Mon, 27 Feb 2023 11:08:49 +0100
+Subject: [PATCH 2/2] test: disable tests involving BlueSquare.gif
+
+The file is GIF87a, not GIF89a, and while we replace the file in a more
+proper commit, we need to have a small patch for Gentoo:
+ https://bugs.gentoo.org/883939
+---
+ test/common_fixtures.py | 2 +-
+ test/samples.py         | 2 +-
+ test/test_exempi.py     | 2 +-
+ 3 files changed, 3 insertions(+), 3 deletions(-)
+
+diff --git a/test/common_fixtures.py b/test/common_fixtures.py
+index 135a795..edbcdd7 100644
+--- a/test/common_fixtures.py
++++ b/test/common_fixtures.py
+@@ -44,7 +44,7 @@ samplefiles = {
+     'BlueSquare.ai'   : libxmp.consts.XMP_FT_ILLUSTRATOR,
+     'BlueSquare.avi'  : libxmp.consts.XMP_FT_AVI,
+     'BlueSquare.eps'  : libxmp.consts.XMP_FT_EPS,
+-    'BlueSquare.gif'  : libxmp.consts.XMP_FT_GIF,
++    #'BlueSquare.gif'  : libxmp.consts.XMP_FT_GIF,
```

Comment:
Don't comment stuff out, just remove it, this will make the patch shorter. Technically, in this case you could even go for `-U0`.

Reply:
OK. I have the habit of usually leaving the comments to document that a change is "dirty". I didn't think about the negative impact on portage.